apiserver/facades/agent/upgradeseries/upgradeseries.go

```diff
@@ -113,6 +113,27 @@ func (a *API) SetMachineStatus(args params.UpgradeSeriesStatusParams) (params.Er
 	return result, nil
 }
 
+// CompleteStatus starts the upgrade series completion phase for all subordinate
+// units of a given machine.
+func (a *API) CompleteUnitUpgradeSeries(args params.SetUpgradeSeriesStatusParams) (params.ErrorResults, error) {
```

Comment:
I would like to tweak the name of this, as mentioned in the earlier commit.

Reply:
That is a good point. The name suggests something other than what it actually does. What is going on is this:

- User does a `juju upgrade-series complete`
- User sees this (since its state was changed by that command) then it does the same for its units (CompleteUnitUpgradeSeries).
- Units run hooks and go to the real `completed` state

So I guess what it really should be called is `StartUnitsCompletePhase` or something like that.